portsyncd/portsyncd.cpp

```diff
@@ -20,6 +20,8 @@
 using namespace std;
 using namespace swss;
 
+#define DEFAULT_SELECT_TIMEOUT 1000 /* ms */
```

Comment:
>1000 [](start = 31, length = 4)

Is it too slow? How about CPU usage @200 ms?

Reply:
Any number of ms should be good here, I don't see a problem with 1000 ms, and this is default for other daemons. Why it is to slow? Which tasks portsyncd needs to do every DEFAULT_SELECT_TIMEOUT ms?

Reply:
From the code, this number is not really determining the frequency of checking the events, since whenever you timeout, you call select again without doing anything, so it is almost always checking the events, thus no slowness concerns.  The code in the review will have similar behavior with bigger timeout even with -1.  But I think setting to 1 is fine.